experiment/cache_exp.cpp

```diff
@@ -20,20 +20,16 @@ static void querier(GutteringSystem *gts) {
   }
 }
 
-static void write_configuration(int queue_factor, int gutter_factor) {
-  std::ofstream conf("./buffering.conf");
-  conf << "queue_factor=" << queue_factor << std::endl;
-  conf << "gutter_factor=" << gutter_factor << std::endl;
-}
-
 static void run_randomized(const int nodes, const unsigned long updates, const unsigned int nthreads=1) {
   shutdown = false;
-  write_configuration(2, 1); // 2 is queue_factor, 1 is gutter_factor
-  CacheGuttering *gutters = new CacheGuttering(nodes, 40, nthreads); // 40 is num workers
+  size_t num_workers = 20;
+
+  GutteringConfiguration conf(1, 20, 64, 8, 2, 1, 8);
```

Comment:
We need to do something to make this more readable.

Reply:
Any suggestions?

Reply:
After thinking it over, the real issue is probably the magic numbers. Should we make global variables to control them? It would also make it easier to see if we're somehow changing a config option from default for a specific test.